openlabcmd/openlabcmd/zk.py

```diff
@@ -297,24 +271,23 @@ def update_service(self, service_name, role, n_type, alarmed=None,
             if not isinstance(alarmed, bool):
                 raise exceptions.ValidationError('alarmed should be boolean '
                                                  'value.')
-            old_service['alarmed'] = alarmed
```

Comment:
Also in here, we update a serivice using a node_type, but the definition of service obj doesn't contain it. How about to add it? as the node_name and node_role are there already and a single service can only match a role:type except the data plane services. Just a nit, as this function can also work

Reply:
I just realized that `node_name` is unique. a service can be fetched by a command like `openlab ha service get <service_name> --node <node_name>` If so, both `node_role` and `node_type` can be removed from `service` obj.

Thought?

Reply:
Correct, let's make it unit function, not coupling so much.